library/SimplePie/Sanitize.php

```diff
@@ -441,7 +511,8 @@ public function replace_urls($document, $tag, $attributes)
 					if ($element->hasAttribute($attribute))
 					{
 						$value = $this->registry->call('Misc', 'absolutize_url', array($element->getAttribute($attribute), $this->base));
-						if ($value !== false)
+						$value = $this->https_url($value);
```

Comment:
`absolutize_url` can return false, I don't think you should check the url in that case.

Reply:
Fixed https://github.com/simplepie/simplepie/pull/676/commits/5bfec781a947654b4f3610690be1572389e93033